buildmaster.R

```diff
@@ -30,6 +30,18 @@ for(k in 1:length(la_all)){
 # la_all <- as.character(regions@data$Region[sel])
 la_all <- la_all[-grep("liv|greater-m", la_all)]
 
+if(grepl("^(.?$|[^n].+)", readline("Would you like to commit the pct-data repo (Y/n)?  "), ignore.case = T)){
+  commitMessage <- readline("Please enter the commit message: ")
+  if(commitMessage = ""){ commitMessage <- "Updating the data, auto commiting"}
```

Comment:
Have you tested this code @nikolai-b ? I'm getting an error message: 

```
> gitDir <- file.path("..", "pct-data", ".git")
> system2("git", c("--git-dir", gitDir, "--work-tree", gitDir,
+ if(grepl("^(.?$|[^n].+)", readline("Would you like to commit the pct-data repo (Y/n)?  "), ignore.case = T)){
+ commitMessage <- readline("Please enter the commit message: ")
+ if(commitMessage = ""){ commitMessage <- "Updating the data, auto commiting"}
Error: unexpected '=' in:
"commitMessage <- readline("Please enter the commit message: ")
if(commitMessage ="
```

Please test your code before committing!